opensearch_py_ml/ml_commons/rag_pipeline/__init__.py

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: Apache-2.0
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+# Any modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
why two times?

Reply:
when I ran nox format it doubled it I guess, the first one is what I put and nox format put the second one. fixed though.